src/taxonomy/import-tags/ImportTagsWizard.jsx

```diff
@@ -216,18 +278,60 @@ const ImportTagsWizard = ({
   taxonomy,
   isOpen,
   onClose,
+  reimport,
 }) => {
   const intl = useIntl();
   const { setToastMessage, setAlertProps } = useContext(TaxonomyContext);
 
-  const steps = ['export', 'upload', 'plan', 'confirm'];
+  const steps = reimport
+    ? ['export', 'upload', 'plan', 'confirm']
+    : ['upload', 'populate', 'confirm'];
+
   const [currentStep, setCurrentStep] = useState(steps[0]);
```

Comment:
nit: this `steps` variable is only used here for this initial `useState(steps[0])`; the code below just checks `reImport` to [decide the next step](https://github.com/openedx/frontend-app-course-authoring/pull/1017/files#diff-f88ee68cfef86847c9a7471bfd24c9764bcdcf086faa10cf9002817bbde5e3daR443).

Do we actually need this `steps` list?

Reply:
You're right, I think it might've related to an older implementation. I've removed it [cddd653](https://github.com/openedx/frontend-app-course-authoring/pull/1017/commits/cddd653b8f87044a5c93e45da61bc7a49881083c)